utils/Felt.go

```diff
@@ -31,6 +31,16 @@ func HexToFelt(hex string) (*felt.Felt, error) {
 	return new(felt.Felt).SetString(hex)
 }
 
+// StrToFelt generates a felt.Felt from a string representation containing ASCII characters, not hex values.
+//
+// Parameters:
+// - str: The string to convert to a felt.Felt
+// Returns:
+// - *felt.Felt: a felt.Felt representing the converted value
+func StrToFelt(str string) *felt.Felt {
+	return new(felt.Felt).SetBytes([]byte(str))
+}
+
```

Comment:
I think this can be deleted as it's not being used. Plus, I feel like it's safer to user SetString when going from string to felt

Reply:
Done